roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -95,7 +95,7 @@ def _print_check_failure_summary(self, failed_checks, context):
             'Variables can be set in the inventory or passed on the\n'
             'command line using the -e flag to ansible-playbook.\n'
         ).format(playbook=self._playbook_file, checks=checks)
-        if context in ['pre-install', 'health']:
+        if context in ['pre-install', 'upgrade', 'health']:
```

Comment:
This is the summary for playbooks where the user is trying to run checks. I wouldn't include "upgrade" here so they can have the previous msg with a little more explanation. If there were a "pre-upgrade" playbook just for running checks then I would put it here.